StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1501CodeFixProvider.cs

```diff
@@ -75,8 +75,19 @@ private static SyntaxNode ReformatBlockAndParent(Document document, SyntaxNode s
                 newParentLastToken = newParentLastToken.WithTrailingTrivia(newTrailingTrivia);
             }
 
+            var parentNextToken = block.CloseBraceToken.GetNextToken();
```

Comment:
:question: What happens if `CloseBraceToken` is the last token in the file? For example, what happens if the following code is the entire file?

```csharp
class TypeName
{
  void MethodName()
  {
    if (false) { return; }
```